compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/TileAttention.cpp

```diff
@@ -145,6 +145,45 @@ static Value insertOutputSlice(Value src, Value dst,
                                     headDimension, loc, builder);
 }
 
+static SmallVector<AffineMap>
+getTileAttentionIndexingMaps(RewriterBase &rewriter, int64_t tiledInputRank,
+                             bool transposeV) {
+  MLIRContext *ctx = rewriter.getContext();
+  AffineExpr m, k1, k2, n;
+  bindDims(ctx, m, k1, k2, n);
+
+  AffineMap qMap =
+      AffineMap::get(/*dimCount=*/4, /*symbolCount=*/0, {m, k1}, ctx);
+  AffineMap kMap =
+      AffineMap::get(/*dimCount=*/4, /*symbolCount=*/0, {k2, k1}, ctx);
+  AffineMap vMap =
+      AffineMap::get(/*dimCount=*/4, /*symbolCount=*/0, {k2, n}, ctx);
+  AffineMap rMap =
+      AffineMap::get(/*dimCount=*/4, /*symbolCount=*/0, {m, n}, ctx);
+  AffineMap maxMap =
+      AffineMap::get(/*dimCount=*/4, /*symbolCount=*/0, {m}, ctx);
+  AffineMap sumMap =
+      AffineMap::get(/*dimCount=*/4, /*symbolCount=*/0, {m}, ctx);
+
+  if (transposeV) {
+    SmallVector<AffineExpr> vDims(vMap.getResults());
+    std::swap(vDims[0], vDims[1]);
+    vMap = AffineMap::get(vMap.getNumDims(), vMap.getNumSymbols(), vDims, ctx);
+  }
+
+  SmallVector<AffineMap> attentionMaps{qMap, kMap, vMap, rMap, maxMap, sumMap};
```

Comment:
nit: Prefer using `()` constructor or `=` assignment instead of brace-list initialization. This kind of code is dangerous in some types. E.g.,

```
std::vector<std::string> strings{2}; // A vector of two empty strings.
std::vector<int> ints{2};            // A vector containing only the integer 2.
```

I'd suggest to use either direct constructer or assignment style for consistency in the codebase.

See https://abseil.io/tips/88 for more details.

so it could be `SmallVector<AffineMap> attentionMaps = {qMap, kMap, vMap, rMap, maxMap, sumMap};`.

Reply:
Thanks for the pro-tip!